src/m365/planner/commands/tenant/tenant-settings-set.ts

```diff
@@ -78,6 +78,15 @@ class PlannerTenantSettingsSetCommand extends PlannerCommand {
     );
   }
 
+  #initTypes(): void {
+    this.types.boolean.push('isPlannerAllowed');
```

Comment:
We can simplify this by pushing multiple values at once, eg. `this.types.boolean.push('isPlannerAllowed', 'allowCalendarSharing', ...) `